docs/source/quick_start.mdx

```diff
@@ -192,11 +200,10 @@ curl https://datasets-server.huggingface.co/is-valid?dataset=rotten_tomatoes \
 </curl>
 </inferencesnippet>
 
-This returns whether the `valid` key is `true` or `false`:
+This returns whether the dataset provides a preview (see /first-rows), the viewer (see /rows) and the search (see /search):
```

Comment:
I don't see the `/search` endpoint documented anywhere so we should probably add that to the docs as well

```suggestion
This returns whether the dataset provides a preview (see `/first-rows`), the viewer (see `/rows`), and whether the dataset is searchable (see `/search`):
```

Reply:
Oh yes, I'll do it in this PR.

Reply:
added